baybe/recommenders/meta/base.py

```diff
@@ -20,6 +20,12 @@
 class MetaRecommender(SerialMixin, RecommenderProtocol, ABC):
     """Abstract base class for all meta recommenders."""
 
+    _current_recommender: PureRecommender | None = field(default=None, init=False)
+    """The current recommender."""
+
+    _has_been_used_for_recommendation: bool = field(default=False, init=False)
```

Comment:
This name is confusing since it is not clear from the name that this refers to the current recommender. Maybe `_current_recommender_was_used`?

Reply:
Also, shouldn't this technically be a field of `PureRecommender`?

Reply:
First point: yes, agree, definitely better. Thx!

Second point: might look like at first glance (and in fact, the `PureRecommender` could *also* have such an attribute if we ever need it on that level), but if you think about it: in this context, the flag is really a property of the meta recommender. Because it matters if the recommender has been already used *by* the meta recommender, and not outside of it. As a counter-example: you could create a pure recommender, use it for several recommendations, and *then* pass that used object to a fresh meta recommender instance as part of its recommender queue. The meta recommender should still process that pure recommender from its queue like all others, regardless of whether it has already been used in other contexts. Makes sense?

I'll resolve, but feel free to reopen if necessary.